### ISSUE TITLE
New release?

### ISSUE THREAD

**Issue by brjoha** [Tencent/rapidjson#1550]
Consider issuing a v1.1.1 release or better in the near future?

The last release is nearly three years old, which makes it look decrepit (and bug riddled given nearly 500 commits since) in the eyes of some corporate 3rd party licensing folks who shall remain unnamed.

**Comment by anthraxx** [comment]
yes, please do. Distros are using a 3 years old version to build dependencies. It would be awesome to have frequent release  tags if useful things changed.

**Comment by wwizz** [comment]
Same here, I have troubles intaking the library since I want to use a more recent version than version 1.1.0. Our build system is setup such that it takes tags (not commit hashes) for including external libs.

**Comment by Croydon** [comment]
This is a duplicate of #1546

**Comment by Livius90** [comment]
Use nlohmann C++ json lib, it is much better in releasing and documentation. I gave up waiting for new rapidjson release, i am using this other from now.
https://github.com/nlohmann/json
